dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -76,10 +76,11 @@ BUILD()
 		autoreconf -vfi
 		cd ./build-$pythonPackage
 
+		runConfigure \
 		../configure PYTHON_VERSION=$pythonVersion \
 			--prefix=$prefix
 
-		make "$jobArgs"
+		make runConfigure "$jobArgs"
```

Comment:
Are you sure here? Haven't seen runConfigure being used with make before ...

Reply:
LEFTOVER. AGAIN. MY BAD.

Reply:
That's leftover code, I talked about how I attempted to do that in the IRC, but I removed it in my development environment and kept it in my IDE. (due to my inexperience with anything make-related that's not a thread parameter)